src/controllers/notification.js

```diff
@@ -0,0 +1,119 @@
+/* eslint-disable require-jsdoc */
+import { newTripNotification } from '../utils/emailHelper';
+import Models from '../database/models';
+import { onlineClients } from '../utils/socket';
+import setLanguage from '../utils/international';
+
+class notificationsController {
+  static async sendNotification(user, type, request, host) {
+    try {
+      const { firstName, managerEmail } = user,
+        {
+          requestId, from, to, reason
+        } = request,
+        { Users } = Models,
+        manager = await Users.findOne({
+          where: {
+            email: managerEmail
+          },
+        }),
+        body = ` ${firstName} made a ${type}: - From ${from} to ${to}. Reason: ${reason}`,
+        notification = {
+          requestID: requestId, receiver: manager.userID, type, body, isRead: false
+        };
+      await Models.Notification.create(notification);
+      if (manager.notifyInApp) {
+        const online = onlineClients.get(manager.userID.toString());
+        if (online) {
+          online.emit('new-trip-request', notification);
+        }
+      }
+      if (manager.notifyByEmail) {
+        await newTripNotification(manager.firstName, manager.email, body, requestId, host);
+      }
+    } catch (error) {
+      return console.log(error);
+    }
+  }
```

Comment:
@RusimbiP Would you please endeavor to return a response to user instead of logging

Reply:
@hakoemmy This method does not take in req and res.

Reply:
@RusimbiP I got the point